FV3/gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -747,6 +748,7 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
     integer              :: iopt_stc  !snow/soil temperature time scheme (only layer 1)
 
     logical              :: use_ufo         !< flag for gcycle surface option
+    logical              :: use_atm_ts_as_sst ! whether to set physics SST to dynamical core ts
```

Comment:
I think I see where you're coming from.  This is perhaps the most literal description of what this namelist parameter does. That said I feel like we should possibly mention "analysis" or "nudging" in the name in some way, something more like Lucas's suggestion, e.g. `use_analysis_sst`.  This would make it clear that this is the option for allowing you to run with the same SSTs as used in the analysis.

The reason being is that I'm not sure how useful `Atm%ts` is outside of the nudging context.  For reference, I have tested what happens when you turn nudging off (i.e. don't read in any nudging data) but use this parameter, and all that occurs is that the SST stays constant and set to the initial condition.

So ultimately what this namelist parameter does is one of two things:
1. Causes the model to use the initial condition SST for the whole simulation if nudging is not active.
2. Causes the model to use the analysis SST if nudging is active.

I'm not sure if either of those two outcomes is obvious from the namelist parameter's current name.

Reply:
Fair enough, maybe I was being a bit too literal here. I'll change it to `use_analysis_sst` and add a note specifying that it will only do that if nudging is active.

It wasn't clear to me what would happen with nudging off and this option on. Good to know (and actually that seems to be the same behavior that is currently happening with the fv3atm public release).

Reply:
Sounds good!